backend_tools/db_provider.ts

```diff
@@ -80,7 +80,14 @@ class DbProvider {
 
   async getClubsByName(name: string): Promise<Club[]> {
     const clubRef = collection(this.db, this.clubPath);
-    const q: Query<DocumentData> = query(clubRef, where('name', '==', name));
+    const capitalizedName = name.charAt(0).toUpperCase() + name.slice(1);
+
+    const q: Query<DocumentData> = query(
```

Comment:
This looks like a great change to support searching with the first character non-capitalized or capitalized. Maybe it would make sense to support searching with any capitalization by performing all searches as if every letter of the name is lowercase. Perhaps we could have an auto-generated "searchName" field in Firebase including a sanitized org name that is easier to query.

One issue I ran into with a quick test is that searching "NEBUL" returns no autocomplete results, while "Nebul" works as expected.

Reply:
Yea, I was gonna try to implement a fuzzy search algorithm but I want to discuss with the team to see if it may cause unnecessary queries

Reply:
I asked some other people and they recommended Fuse.js so I'll look into that